lib/src/dart_dev_cli.dart

```diff
@@ -63,11 +64,27 @@ dev(List<String> args) async {
   registerTask(new InitCli(), config.init);
   registerTask(new TestCli(), config.test);
 
+  try {
+    LocalCli.discover(config.local.taskPaths).forEach((task, exec) {
+      registerTask(new LocalCli(task, exec), config.local);
+    });
```

Comment:
@georgelesica-wf @evanweible-wf I moved the discovery functionality into a static function that can be used to register each local task as it's own LocalCli object which effectively registers and executes the same as before.